app/components/node-navbar/template.hbs

```diff
@@ -0,0 +1,41 @@
+{{#if renderInPlace}}
+    <div id='rightHere'></div>
+{{/if}}
+{{#ember-wormhole to=(if renderInPlace 'rightHere' 'secondaryNavbar')}}
+    <nav class='navbar osf-project-navbar' role='navigation'>
+        <div class='container'>
+            <div class='navbar-header'>
+                <button type='button' class='navbar-toggle collapsed' data-toggle='collapse' data-target='.project-nav'>
+                    <span class='sr-only'>{{t 'node_navbar.toggle'}}</span>
+                    <span class='fa fa-bars fa-lg'></span>
+                </button>
+                <span class='navbar-brand visible-xs visible-sm'>
+                    {{t 'node_navbar.project_nav'}}
+                </span>
+            </div>
+            <div class='collapse navbar-collapse project-nav'>
+                <ul class='nav navbar-nav'>
+                    <li class="{{if (eq active 'title') 'active'}}"><a href='/{{node.id}}/' class='project-title'>{{node.title}}</a></li>
```

Comment:
Why are these not link-to's?

Reply:
Because we will be implementing the node sub-pages (such as forks and registrations) before we have every page done. It seems like it would make more sense to change them to link-to's when we implement those pages.